microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -70,7 +70,7 @@ If you have a Partner Center account for which the Primary Owner is a Microsoft
 
 1.  After you sign in, a registration form is displayed, to enroll in the Microsoft Edge program.  To help you fill in the registration form, see the next section.
 
-1.  Before submitting your registration form, read and accept the terms and conditions of the [Microsoft Edge Developer Agreement](/legal/windows/agreements/app-developer-agreement).
+1.  Before submitting your registration form, read and accept the terms and conditions of the [Microsoft Store App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922).
```

Comment:
globally in PR / in repo, i plan to change to /legal/... path instead of fwlink, based on research shared w/ @captainbrosset -
* Legal repo defines redirect from /legal/windows/agreements/app-developer-agreement to https://go.microsoft.com/fwlink/?linkid=528905 
* The redirect in Legal repo is 528905 not 221922.
* Purpose of fwlinks: use to link from product code base, quickly updatable outside of the product code base.
* Standardize how links are tagged in repo, for easiest maintenance (& accuracy).
* Repo can use fwlinks for special external Microsoft sites, esp Support, & Edge Preview channels site.  Not for articles that are at LMC.

![image](https://github.com/user-attachments/assets/040a46d0-c8cf-4ca9-8b28-4b2c3daadc43)

@bhuvanapriyap @sivMSFT